include/props.h

```diff
@@ -54,123 +77,1286 @@ typedef struct plist *PropPtr;
 /* Blessed props evaluate with wizbit MPI perms. */
 #define PROP_BLESSED     0x1000
 
+/* You will never want to change these, or you will make your MUCK
```

Comment:
Aren't these here for future internationalization?  I have no idea, I'm just guessing, so, @wyld-sw could you elaborate?  I know you've done some stuff related to this.

Reply:
They might be there to be mnemonic/useful instead of magic symbols, I guess. >w>